### PR TITLE
combine recipe

### PR DESCRIPTION
**Pull request by DrWelles**

### CODE REVIEW

File: egs/combine_data/svs1/local/data.sh [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+# fi
+
+
+# if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
+#     log "stage 1: Dataset split "
+#     python local/dataset_split.py ${KIRITAN}/kiritan_singing/wav data/local 0.1 0.1
+# fi
+
+for dataset in ${train_set} ${train_dev} ${recog_set}; do
```
Comment (ftshijt) [inline egs/combine_data/svs1/local/data.sh:68]: Can you add some notes for users to better understand the stages?

Comment (DrWelles) [reply egs/combine_data/svs1/local/data.sh:56]: Done

File: egs/kiritan/svs1/run.sh [hunk 2]
```diff
@@ -54,5 +54,9 @@ cleaner=none
...
     --test_sets "${test_sets}" \
     --srctexts "data/${train_set}/text" \
+<<<<<<< HEAD
     --local_data_opts "${NOWPATH}/"\
-    ${opts} "$@"
\ No newline at end of file
+    ${opts} "$@"
+=======
+    ${opts} "$@"
+>>>>>>> fdfbfedc87702a68ec427f862ac398153d02d20d
```
Comment (ftshijt) [inline egs/kiritan/svs1/run.sh:62]: Please fix the conflict...

File: egs/ofuton_p_utagoe_db/svs1/cmd.sh [hunk 3]
```diff
@@ -28,7 +28,11 @@
 
 
 # Select the backend used by run.sh from "local", "stdout", "sge", "slurm", or "ssh"
+<<<<<<< HEAD
```
Comment (ftshijt) [inline egs/ofuton_p_utagoe_db/svs1/cmd.sh:31]: same here

**Review by ftshijt — CHANGES_REQUESTED** [review]
Many important files are deleted! Please check them carefully before making a PR. This is super dangerous!!!

**Comment by ftshijt** [comment]
THe makefile is still deleted, please recover it

**Comment by ftshijt** [comment]
Also conflict exists

**Comment by ftshijt** [comment]
Can you change the `activate_python.sh` into default version here https://github.com/SJTMusicTeam/Muskits/blob/9eda7fa8dd7a38976cf156a0e2a0175305a2a9dd/tools/activate_python.sh

File: egs/combine_data/svs1/local/dataset_split.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
```
Comment (ftshijt) [inline egs/combine_data/svs1/local/dataset_split.py:1]: ditto

File: egs/combine_data/svs1/local/prep_segments.py [hunk 5]
```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
```
Comment (ftshijt) [inline egs/combine_data/svs1/local/prep_segments.py:1]: ditto

**Comment by ftshijt** [comment]
Can you also add a readme file or some explanation in the recipe, so that helps others to get start?

**Pull request merged**